lib/swagger_yard/type.rb

```diff
@@ -77,13 +77,31 @@ def to_h
       if array?
         { "type" => "array", "items" => Type.from_type_list([array.join("<")]).to_h }
       elsif object?
-        {
-          "type" => "object",
-          "additionalProperties" => Type.from_type_list([object.join("<")]).to_h
-        }
+        parse_object
       else
         type
       end
     end
+
+    def parse_object
+      properties = {}
+      additional = nil
+      type_hash  = { "type" => "object" }
+
+      object.join("<").split(/,\s?/).each do |item|
+        key, *rest = item.split(": ")
+
+        # TODO: Should we raise an error if more than one additional type?
```

Comment:
I think that disallowing more than one additional property type (best by raising an appropriate error that is similar to other parsing issues) would be a great idea here, given that the JSON Schema specs do not seem to allow for more than one additional property type.